lib/c-api/src/wasm_c_api/unstable/parser/operator.rs

```diff
@@ -0,0 +1,1017 @@
+use wasmer::wasmparser::Operator;
```

Comment:
Clearly this file should be produced by a proc-macro that runs over wasmparser.

No action required. Please.

Reply:
🤣